pages/common/updog.md

```diff
@@ -0,0 +1,25 @@
+# updog
+
+> A Replacement for Python's SimpleHTTPServer.
+> It allows uploading and downloading via HTTP/S, can set ad hoc SSL certificates and use HTTP basic auth.
+> More information: <https://github.com/sc0tfree/updog>.
+
+- Start a HTTP server on your current directory:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Start a HTTP server for the current directory:
```